faucet/evm.go

```diff
@@ -25,7 +25,7 @@ type EVM struct {
 	// chainID chainId/networkId of the network
 	chainID int
 	// amount of tokens to be transferred
-	amount uint64
+	amount *big.Int
```

Comment:
We are using uint64 amounts for the vocdoni-node Account implementation so I won't change this type.